{{cookiecutter.project_name}}/examples/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.10.2)
+project({{ cookiecutter.module_name }}Examples)
```

Comment:
Also, do we need to enable and add tests for the examples?

Reply:
If the example does something, we can use it to write a test, see [this](https://github.com/KitwareMedical/ITKMorphologicalContourInterpolation/blob/dcae5d5b693098a98236d21e5191baa2f8abf74b/examples/CMakeLists.txt#L18-L23). Having test data in there is harder to properly support, that's why [I removed it](https://github.com/KitwareMedical/ITKMorphologicalContourInterpolation/commit/716b4a3ccbb329e54d9e2bd1d1d914699e1ccb1b).